lib/CL/pocl_binary.c

```diff
@@ -52,7 +52,10 @@
 #define POCLCC_STRING_ID_LENGTH 8
 /* changes for version 2: added program.bc right after header */
 /* changes for version 3: added flush_denorms flag into header */
-#define POCLCC_VERSION 3
```

Comment:
I don't think we need to gracefully handle previous pocl binary versions before we have 1.0 out. But OK.